spec/value_objects/challenged_decision_status_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+RSpec.describe ChallengedDecisionStatus do
```

Comment:
I don't think we really need to test this – we're not doing it for the other value objects either.